source/frontend/src/interfaces/IContactSearchResult.ts

```diff
@@ -27,59 +19,107 @@ export interface IContactSearchResult {
   tenantType?: string;
```

Comment:
no need to change now, but these should not be optional, but rather allow for undefined or null instead